.travis.yml

```diff
@@ -36,10 +36,13 @@ jobs:
       before_deploy:
         - poetry config repositories.mdsol https://mdsol.jfrog.io/mdsol/api/pypi/pypi-local
```

Comment:
@jfeltesse-mdsol Do we still need to publish to Artifactory? I think publishing just to PyPI is enough.

Reply:
agreed, PyPI is enough.